### PR TITLE
renaming filter as custom, converter as auto

### PR DESCRIPTION
**Pull request by yoouung**
- rename filter as customFilter, and converter as autoFilter

### CODE REVIEW

File: dt-demo-be/app_lab.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 test_doc = None # This is user-uploaded file
 doc_list = []
 
-@app.route("/api/filter/upload", methods=["POST"])
+@app.route("/api/customfilter/upload", methods=["POST"])
 def file_upload():
```
Comment (doorooful) [inline dt-demo-be/app_lab.py:13]: Can you rename this function as custom_file_upload
to keep consistency?

File: dt-demo-be/app_lab.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def post_selected_modules():
 
     return jsonify({'result on setting active modules': result})
 
-@app.route("/api/filter/filelist", methods=["GET"])
+@app.route("/api/customfilter/filelist", methods=["GET"])
 def file_list():
```
Comment (doorooful) [inline dt-demo-be/app_lab.py:43]: Same here!

File: dt-demo-fe/src/components/Navbar/index.js [hunk 3]
```diff
@@ -11,11 +11,11 @@ const Navbar = () => {
...
                     </NavLink>
-                    <NavLink to="/filter" activeStyle>
-                        Filter
+                    <NavLink to="/customfilter" activeStyle>
+                        커스텀 필터링
                     </NavLink>
-                    <NavLink to="/converter" activeStyle>
-                        Converter
+                    <NavLink to="/autofilter" activeStyle>
+                        오토 필터링
```
[the remark this answers is not in the document]
Comment (doorooful) [reply dt-demo-fe/src/components/Navbar/index.js:18]: 👀

File: dt-demo-fe/src/pages/autoFilter/conversionReport.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const AutoConversion = () => {
```
Comment (doorooful) [inline dt-demo-fe/src/pages/autoFilter/conversionReport.js:1]: How about change it into AutoFilterConversion?

File: dt-demo-fe/src/pages/autoFilter/conversionReport.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+    return <div>autofilter 활용하여 생성한 결과</div>;
+};
+
+export default AutoConversion;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/autoFilter/conversionReport.js:5]: Same here

File: dt-demo-fe/src/pages/autoFilter/index.js [hunk 6]
```diff
@@ -10,13 +10,13 @@ const Container = styled.div`
     justify-content: center;
 `;
 
-const ConverterMain = () => {
+const AutoMain = () => {
```
Comment (doorooful) [inline dt-demo-fe/src/pages/autoFilter/index.js:13]: I think we could name this as AutoFilterMain regarding other variable names

File: dt-demo-fe/src/pages/autoFilter/index.js [hunk 7]
```diff
@@ -10,13 +10,13 @@ const Container = styled.div`
...
         <Container>
-            <FileListContainer mode="converter" />
+            <FileListContainer mode="autofilter" />
             <RealtimeConverter />
         </Container>
     );
 };
 
-export default ConverterMain;
+export default AutoMain;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/autoFilter/index.js:22]: Same here

File: dt-demo-fe/src/pages/customFilter/conversionReport.js [hunk 8]
```diff
@@ -28,7 +28,7 @@ const BottomContainer = styled.div`
     align-items: center;
 `;
 
-const ConversionContainer = () => {
+const CustomConversion = () => {
```
Comment (doorooful) [inline dt-demo-fe/src/pages/customFilter/conversionReport.js:31]: Can you name it as CustomFilterConversion?

File: dt-demo-fe/src/pages/customFilter/conversionReport.js [hunk 9]
```diff
@@ -76,4 +76,4 @@ const ConversionContainer = () => {
     );
 };
 
-export default ConversionContainer;
+export default CustomConversion;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/customFilter/conversionReport.js:79]: Same here

File: dt-demo-fe/src/pages/customFilter/detectionReport.js [hunk 10]
```diff
@@ -39,7 +39,7 @@ const RadioGroup = styled.div`
     width: 500px;
 `;
 
-const DetectionContainer = () => {
+const CustomDetection = () => {
```
Comment (doorooful) [inline dt-demo-fe/src/pages/customFilter/detectionReport.js:42]: It'd be better to use CustomFilterDetection

File: dt-demo-fe/src/pages/customFilter/detectionReport.js [hunk 11]
```diff
@@ -57,13 +57,13 @@ const DetectionContainer = () => {
         }
 
         try {
-            const convReport_data = await useApi.post("/filter/convreport", {
+            const convReport_data = await useApi.post("/customfilter/convreport", {
```
Comment (doorooful) [inline dt-demo-fe/src/pages/customFilter/detectionReport.js:60]: Can you also confirm detReport_data? I think the api url is not changed there

File: dt-demo-fe/src/pages/customFilter/detectionReport.js [hunk 12]
```diff
@@ -141,4 +141,4 @@ const DetectionContainer = () => {
     );
 };
 
-export default DetectionContainer;
+export default CustomDetection;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/customFilter/detectionReport.js:144]: It'd be better to use CustomFilterDetection

File: dt-demo-fe/src/pages/customFilter/index.js [hunk 13]
```diff
@@ -10,13 +10,13 @@ const Container = styled.div`
     justify-content: center;
 `;
 
-const FilterMain = () => {
+const CustomMain = () => {
```
Comment (doorooful) [inline dt-demo-fe/src/pages/customFilter/index.js:13]: Can you change it into CustomFilterMain?

File: dt-demo-fe/src/pages/customFilter/index.js [hunk 14]
```diff
@@ -10,13 +10,13 @@ const Container = styled.div`
...
         <Container>
-            <FileListContainer mode="filter" />
+            <FileListContainer mode="customfilter" />
             <RulesContainer />
         </Container>
     );
 };
 
-export default FilterMain;
+export default CustomMain;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/customFilter/index.js:22]: Same here!

**Review by doorooful — CHANGES_REQUESTED** [review]
Please refer to my comments!
I think it'd be better to choose only one between auto/custom or autofilter/customfilter.
- option1: Follow comments
I'd just commented upon Flask api url, which was changed from filter->customfilter, converter->autofilter.

- option2: Another way
If you'd choose custom/auto then please change api endpoint and related var names as not to contain filter(i.e. /api/custom/upload- o, /api/customfilter/upload - x)

File: dt-demo-fe/src/components/Navbar/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (yoouung) [reply dt-demo-fe/src/components/Navbar/index.js:18]: applied

File: dt-demo-fe/src/pages/customFilter/detectionReport.js (hunk 11 above)
Comment (yoouung) [reply dt-demo-fe/src/pages/customFilter/detectionReport.js:60]: there's no detection report in auto filtering, so I think it doesn't matter

Comment (yoouung) [reply dt-demo-fe/src/pages/customFilter/detectionReport.js:60]: applied :) thanks!!

**Review by doorooful — APPROVED** [review]
Thank you!

**Pull request merged**